### PR TITLE
1 initial setup

### PR DESCRIPTION
**Pull request by SophieBauernfeind**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>${hibernate.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>at.fh.hagenberg.aist.jfuck</groupId>
+            <artifactId>general</artifactId>
+            <version>${jfuck.version}</version>
```
Comment (oliver-krauss) [inline core/pom.xml:25]: Was renamed into Jack for obvious reasons -> https://github.com/FHOOEAIST/jack  are you working on an old version=

File: deployment/src/test/java/science/aist/sushiya/deployment/it/ExampleRestTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+@ContextConfiguration(locations = {"classpath*:controllerConfigTest.xml"})
+public class ExampleRestTest extends AbstractTestNGSpringContextTests {
```
Comment (oliver-krauss) [inline deployment/src/test/java/science/aist/sushiya/deployment/it/ExampleRestTest.java:20]: If not already done in the future pull requests, please create yourself an issue to remove the sample code classes in the future

File: external/languageServer/src/main/java/science/aist/sushiya/languageserver/AbstractLanguageServer.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+import org.eclipse.lsp4j.services.TextDocumentService;
+import org.eclipse.lsp4j.services.WorkspaceService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * @author lhein
+ *
```
Comment (oliver-krauss) [inline external/languageServer/src/main/java/science/aist/sushiya/languageserver/AbstractLanguageServer.java:28]: As this is adapted form a sample - please add a link to the source

File: external/languageServer/src/main/java/science/aist/sushiya/languageserver/AbstractLanguageServer.java [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package science.aist.sushiya.languageserver;
+
+import org.eclipse.lsp4j.services.TextDocumentService;
+import org.eclipse.lsp4j.services.WorkspaceService;
+import org.slf4j.Logger;
```
Comment (oliver-krauss) [inline external/languageServer/src/main/java/science/aist/sushiya/languageserver/AbstractLanguageServer.java:21]: Please create an issue to replace with seshat

File: persistence/pom.xml [hunk 5]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+            
+  
+
```
Comment (oliver-krauss) [inline persistence/pom.xml:5]: I think something went very wrong with the formatter here. Please remove the empty lines

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+            <url>https://aist.fh-hagenberg.at/nexus/repository/maven-snapshots/</url>
+        </snapshotRepository>
+        <site>
+            <id>javadoc</id>
+            <url>dav:https://aist.fh-hagenberg.at/nexus/repository/javadoc/sushiya/${project.version}</url>
+        </site>
+    </distributionManagement>  <!-- variables -->
+    <properties>    <!-- aist dependencies -->
+        <seshat.version>1.1</seshat.version>
+        <jfuck.version>1.1</jfuck.version>    <!-- further dependencies -->
```
Comment (oliver-krauss) [inline pom.xml:29]: see comment above to jfuck -> https://github.com/FHOOEAIST/jack

Comment (oliver-krauss) [reply pom.xml:29]: @pointan looks like we need to exchange this in the Archetype with the public maven dependency

**Review by oliver-krauss — COMMENTED** [review]
Note: @aschuler84  and I both have been beyond lazy with reviewing your work. Feel free to change this in the Toplevel PR instead of here

File: pom.xml (hunk 6 above)
Comment (pointan) [reply pom.xml:29]: Jup, seems like this hasn't been touched for a while. As jfuck has been renamed to jack, one year ago.

**Comment by oliver-krauss** [comment]
Closing as all of this is addressed in the #12 pull request

**Pull request closed without merging**